apps/wallet-mobile/src/TxHistory/TxHistoryList/TxHistoryListItem.tsx

```diff
@@ -123,9 +125,11 @@ const Amount = ({wallet, transaction}: {wallet: YoroiWallet; transaction: Transa
   return (
     <View style={styles.amount} testID="transactionAmount">
       <Text style={style} secondary={transaction.assurance === 'PENDING'}>
-        <Text>{isPrivacyOff ? '*' : formatTokenInteger(asQuantity(amount), wallet.primaryToken)}</Text>
```

Comment:
privacy issue here no? missing `privacyPlaceholder` ?

Reply:
nice catch, yap, need to update it